Assets/Scripts/Models/Events/HeadlineGenerator.cs

```diff
@@ -40,7 +33,7 @@ public void AddHeadline(string headline, bool displayImmediately = true, bool ke
     {
         if (keepInQueue)
         {
-            headlines.Add(headline);
+            PrototypeManager.Headline.Add(new Headline(headline));
```

Comment:
I'm not sure about the pattern here. Should we add (and or modify) things to the Prototype? Aren't the Prototype suppose to be a read only in memory version of what is in the xml definition files?